fsrs/optimizer.py

```diff
@@ -170,7 +170,7 @@ def _compute_batch_loss(self, parameters: list[float]) -> float:
                         y_retrievability, dtype=torch.float64
                     )
 
-                    if card.state == State.Review:
+                    if card.last_review and (x_date - card.last_review).days > 0:
```

Comment:
Does it consider rollover? I mean something like `next_day_start_at` of Anki.

Reply:
> Does it consider rollover?

Currently no, but that could potentially be a cool feature to add in the future. Maybe in a future release, we could add a `day_starts_at` attribute or something to the `Scheduler`.

However, retrievability is currently calculated like this https://github.com/open-spaced-repetition/py-fsrs/blob/816c9d628b04ea3de9601663b8c481c4c854d0e8/fsrs/fsrs.py#L232
meaning that it's simply equal to 1 when elapsed days are 0 (same-day review).

This pr is just trying to make the optimizer consistent with how retrievability is computed when `elapsed_days=0`.